src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -195,4 +208,18 @@ class Docker implements Serializable {
         }
     }
 
+    @NonCPS
+    private Boolean shouldUseTagForce() {
```

Comment:
Prefer this be packaged as a Java method in the plugin.

Reply:
Not sure how to do that, but I can have a look.